builders/interoperability/xcm/remote-evm-calls.md

```diff
@@ -110,7 +110,7 @@ For example, from the relay chain, the [`DescendOrigin`](https://github.com/pari
   DescendOrigin: {
     X1: {
       AccountId32: {
-        network: 'Westend',
+        westend: null,
```

Comment:
it works as it was, but with this change the network isn't filled in correctly. reference encoded call data: 0x630303040b0101000c36e9ba26fa63c60ec728fe75fe57b86a450d94e7fee7f9f9eddd0d3f400d670000

Reply:
That should not be right. This is what or how the origin chain builds the Descend Origin instruction.

This is the XCM message Moonbase receives via UMP/DMP:

```
yarn xcm-decode-para --w wss://wss.api.moonbase.moonbeam.network --b 4430157 --channel dmp

...
Descend Origin:
{"descendOrigin":{"x1":{"accountId32":{"network":{"westend":null},"id":"0x6e3a195f4a67e2e6e96cd5f2070f73f6e66068db3dcbee136c7f3356523c4c76"}}}}
```

So even if you build the instruction in the Relay chain, this is how it is received on the parachain

Reply:
ok then, you would have to change this to `{"network": "westend":null}` instead of `"westend:null"`?

Reply:
Yeah ok good point, I've addressed this